pkg/agent/config.go

```diff
@@ -345,7 +345,7 @@ func InitAgentCmdFlags(c *cobra.Command, cfg *AgentCmdFlags) {
 		false,
 		"Enables the MachineHub mode. The agent will push data to CyberArk MachineHub.",
 	)
-	c.PersistentFlags().MarkHidden("machine-hub")
+	_ = c.PersistentFlags().MarkHidden("machine-hub")
```

Comment:
It might be better to panic if there's an error to avoid the hidden flags getting out of sync with the actual flag names.

Reply:
Good point. I'll fix that.